README.md

```diff
@@ -66,8 +66,39 @@ app:timeout="500"
             app:timeout="500"
             app:texts="@array/examples" />
 ```
+### Updating the view dynamically - Kotlin
+To set the text dynamically, you can use
+
+```kotlin
+val texts = arrayOf("text1", "text2", "text3")
+val fadingTextView = findViewById<FadingTextView>(R.id.fadingTextView)
+fadingTextView.setTexts(texts) //You can use an array resource or a string array as the parameter
```

Comment:
Please add a space after the `//`, this should happen automatically when you run android studio's auto format

Reply:
I actually meant in all cases but I see now that it's inconsistent in the entire file so it's not pressing